pymess/backend/dialer/__init__.py

```diff
@@ -67,10 +71,11 @@ def _update_dialer_states(self, messages):
 
     def bulk_check_dialer_status(self):
         """
-        Method that finds messages that are not in the final state and updates their states.
+        Method that finds messages that are not in the final state which were not sent and updates their states.
         """
         messages_to_check = self.model.objects.filter(
             is_final_state=False,
+            sent_at__isnull=False,
```

Comment:
the sync would fail otherwise on records which are not in Daktela admin (testing orders already on our accept server)